Automattic/jetpack
# Settings: Newsletter: Adding connection banner to each section

Fixes https://github.com/Automattic/jetpack-marketing/issues/1021

## Proposed changes:

* adding user connection banner to each section of the Newsletters Settings group

Updated panels:
![SCR-20240930-ootz](https://github.com/user-attachments/assets/36e4c3a9-1db7-4bc8-9a47-355a8a3c7062)


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create a JN instance from this branch
* Connect the site but not the user
* Go to `Settings/Newsletter` and verify that each section has a banner informing about user connection
* Connect a user and verify that the banner is not visible
